pkgs/development/tools/misc/deheader/default.nix

```
@@ -0,0 +1,64 @@
+{ lib, stdenv, python3, xmlto, docbook-xsl-nons, fetchFromGitLab }:
```

```suggestion
{ lib
, stdenv
, python3
, xmlto
, docbook-xsl-nons
, fetchFromGitLab
}:
```

For cleaner diffs in case inputs change.